stages/org.osbuild.greenboot

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+"""
+Configure greenboot
+
+Update configuration of greenboot in /etc/greenboot/greenbot.conf.
+"""
+
+import sys
+
+import osbuild.api
+
+
+SCHEMA_2 = r"""
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "monitor_services": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+    }
+  }
```

Comment:
This should be in `options`, not `input`. I am actually surprised (and a bit shocked) that this works.
Also I would say lets move all that into a `config` section, so that in the future, we could maybe have a top level `filename` directive if we move the config around (say to `/usr` or something).

Reply:
fixed